tests/test_multi_gateway.py

```diff
@@ -4,8 +4,8 @@
 import json
 import time
 from control.server import GatewayServer
-from proto import gateway_pb2 as pb2
-from proto import gateway_pb2_grpc as pb2_grpc
+from ..proto import gateway_pb2 as pb2
```

Comment:
@epuertat, this test is failing with:
` from control.server import GatewayServer
>   from ..proto import gateway_pb2 as pb2
E   ImportError: attempted relative import beyond top-level package`
Could you please take a look?